README.md

```diff
@@ -165,53 +165,17 @@ A list of sample names is also available in the `samples` attribute of the parse
 
 ## Breakends
 
-If a variant line has `SVTYPE=BND`, the `ALT` field will be examined for breakend
-specifications, and those will be parsed as objects.  For example:
+We offer a helper function to parse breakend strings. We used to parse these
+automatically but it is now a helper function
 
-```text
-13	123456	bnd_U	C	C[2:321682[,C[17:198983[	6	PASS	SVTYPE=BND;MATEID=bnd V,bnd Z
-```
+import {parseBreakend} from '@gmod/vcf' and call e.g.
 
-will be parsed as
-
-```json
-{
-  "CHROM": "13",
-  "POS": 123456,
-  "ID": [
-    "bnd_U"
-  ],
-  "REF": "C",
-  "ALT": [
-    {
-      "MateDirection": "right",
-      "Replacement": "C",
-      "MatePosition": "2:321682",
-      "Join": "right"
-    },
-    {
-      "MateDirection": "right",
-      "Replacement": "C",
-      "MatePosition": "17:198983",
-      "Join": "right"
-    }
-  ],
-  "QUAL": 6,
-  "FILTER": "PASS",
-  "INFO": {
-    "SVTYPE": [
-      "BND"
-    ],
-    "MATEID": [
-      "bnd V",
-      "bnd Z"
-    ]
-  }
-}
+```
+parseBreakend('C[2:321682[')
```

Comment:
Could we keep an example of what a parsed breakend looks like in the README?

Reply:
yep, added the output of parseBreakend inside this code snippet